web_app/contract_tools/blockchain_call.py

```diff
@@ -352,5 +352,37 @@ async def add_extra_deposit(self, contract_address: str, token_address: str, amo
             calldata=[self._convert_address(token_address), amount],
         )
 
+    async def withdraw_all(self, contract_address: str) -> dict[str, str]:
+        """
+        Withdraws all supported tokens from the contract by calling withdraw with amount=0.
+        
+        :param contract_address: The contract address to withdraw from
+        :return: A dictionary summarizing the results for each token.
+        """
+        contract_addr_int = self._convert_address(contract_address)
+        results = {}
+
+        for token in TokenParams.tokens():  
+            try:
+                token_addr_int = self._convert_address(token.address)  
+                token_symbol = token.name
+                logger.info(f"Withdrawing {token_symbol} from contract {contract_address}")
+                await self._func_call(
+                    addr=contract_addr_int,
+                    selector="withdraw",
+                    calldata=[token_addr_int, 0] 
+                )
+                results[token_symbol] = "Success"
+
+            except ValueError as e:
```

Comment:
this value error only for `self._convert_address(token.address)  `
why rest code is doing in this try except?